tsfc/finatinterface.py

```diff
@@ -122,6 +122,12 @@ def convert_finiteelement(element, **kwargs):
             lmbda = finat.Lagrange
         elif kind == 'spectral' and element.cell().cellname() == 'interval':
             lmbda = finat.GaussLobattoLegendre
+        elif kind in ['mgd', 'feec', 'qb', 'mse']:
```

Comment:
Would it make sense to instead make new UFL elements?  What would they be called?

Reply:
Problem is that these are things like `'spectral'` (GLL / GL) or equispaced Lagrange, so for each you would need names for continuous, discontinouos, H(div), and H(curl).

Reply:
Ah, OK.